src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderTypeAheadMenu.tsx

```diff
@@ -98,14 +99,18 @@ export class GlobalHeaderTypeAheadMenu extends React.Component<Props, State> {
       />
     )
 
+    // max height should be 150, so we take the min of calculated and 150
+    const calculateDropdownHeight = () =>
+      Math.min(queryResults.length * this.listItemHeight, 150)
```

Comment:
since this is a class componet, it's probably better to make this a private instance method on the class that takes the `queryResults.length` as an argument, rather than re-creating the function on every render call. I think it's also a good idea to make a variable to store the value `150` into so it provides context and isn't a magic number (even though it's probably fine since the comment explains what 150 means.

Reply:
I agree, I fixed those things 👍